starwars/src/App.js

```diff
@@ -27,6 +28,29 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <div className="flexContainer">
+          {this.state.starwarsChars.map(data => {
+            return (
+              <div className="flexContainer__card">
```

Comment:
Let me guess. You're getting a warning about react keys? Be sure to include a `key={foo.prop}` on the outer most element that you're mapping to the dom.